papers/MolCLR_2024.md

```diff
@@ -0,0 +1,101 @@
+
+![Figure taken from the blog post by Ekin Tiu illustrating the idea behind contrastive learning.](MolCLR_2024_images/Constrative learning .png)
+
+## NT-Xent loss
+
+The authors use a NT-Xent loss to train the graph encoders.
+
+$$
+L_{ij} = -\log \left( \frac{\exp(\text{sim}(z_i, z_j)/\tau)}{\sum_{k=1}^{2N} \mathbf{1}_{\{k \neq i\}} \exp(\text{sim}(z_i, z_k)/\tau)} \right)
+$$
+
+- `z_i` and `z_j`: Latent vectors extracted from a positive data pair.
+- `N`: Batch size, indicating the number of data pairs used.
+- `sim(⋅)`: Function measuring the similarity between two vectors.
+- `τ`: The temperature parameter, used to scale the similarity measures in the function.
+
+To get more understanding, there is literature paper on contrastive learning one shoould read [@le2020contrastive].
+
+## Overview of MolCLR framework
+
+- The SMILES are converted into graphs in a batch. The graphs consists of nodes and edges. Nodes represents atoms and edges represents chemical bonds of a molecule.
+
+- The authors studied three types of augmentations like atom masking, bond deletion and sub-graph removal.
+
+- In the current work, they used sub-removal with some probability 0.25. Now, the model tries to learn hidden patterns of the molecule within the remaining subgraphs.
+
+- The GNNs works on message passing framework, which means at each node the local neighborhood information is aggregated and updated iteratively. Finally, we obtain embeddings that poses every information about the molecule.
+
+- This embeddings are fed forwarded to the MLP (Multi Layer preceptron) with hidden units to get latent representation of the molecule.
+
+- The loss for the similar representations should be minimized and maximized for the disimilar representations.
+
+![Overview figure for the MolCLR framework](MolCLR_2024_images/Overview figure MolCLR.png)
+
+For more information about GNNs and its applications check out this review paper [zhou2020graph].
+
+## Results
+The authors tested the performance of MolCLR framework on seven benchmarks for classification task and six benchmarks for regression task.
+
+The authors claim that on classification task with other self-supervised learning or pre-training strategies, their MolCLR framework achieves the best performance on five out of seven benchmarks, with an average improvement of 4.0 percentage.
+
+![Table 1 shows the test performance on seven benchmarks on classifiaction task](MolCLR_2024_images/classifion benchmark.png) 
+
+The authors also claim that on regression tasks, MolCLR surpasses other pre-training baselines in five out of six benchmarks and achieves almost the same performance on the remaining ESOL benchmark. For QM9 dataset, MolCLR does not have comparable performable with SchNet and MGCN. As these two models are specifically designed for quantum interaction and make use of extra 3D positional information.
```

Comment:
```suggestion
The authors also claim that on regression tasks, MolCLR surpasses other pre-training baselines in five out of six benchmarks and achieves almost the same performance on the remaining ESOL benchmark. For the QM9 dataset, MolCLR does not have comparable performable with SchNet and MGCN. As these two models are specifically designed for quantum interaction and make use of extra 3D positional information.
```

- "does not have comparable performable with" is awkward 
- what do you mean with "designed for quantum interaction"?